infrastructure/prod.tfvars

```diff
@@ -1,2 +0,0 @@
-capacity = "2"
```

Comment:
No, I will fix that!

Reply:
Actually this was correct -- I thought it was the file that was deleted
We only want 1 instance of the API running, as part of the strategy to prevent duplicate emails being sent. Otherwise CCD could send 2 callbacks, one to each instance and we would not be able to prevent 2 emails being sent.

Reply:
And the default capacity is `1`